WordPress/build.gradle

```
@@ -501,9 +500,23 @@ task violationCommentsToGitHub(type: se.bjurr.violations.comments.github.plugin.
     ]
 }
 
-task printVersionName {
+task "updateVersionProperties" {
     doLast {
-        println android.productFlavors.vanilla.versionName
+        ant.propertyfile(
+                file: "../version.properties") {
+            entry( key: "$key", value: "$value")
+        }
+    }
+}
+
+task "printVersionName" {
```

```suggestion
tasks.register("printVersionName") {
```

https://docs.gradle.org/current/userguide/task_configuration_avoidance.html

---

I don't have a solid advice on this, but the `$VERSION_NAME` in CircleCI and the `printVersionName` is pretty confusing to me because they use the same naming but refer to different things.